supplementary_style_guide/master.adoc

```diff
@@ -164,6 +164,9 @@ include::glossary_terms_conventions/product_conventions/products_conventions_ove
 ==== Azure and .NET Core
 include::glossary_terms_conventions/product_conventions/azure-dotnet.adoc[leveloffset=+1]
 
+==== Partner and Hybrid Cloud Infrastructure
```

Comment:
@laubai Can you share a bit more about the intent of this new section? Per your commit msg, I was thinking it was mostly going to be storage-related (which is probably is), but there are also a lot of other general terms in here as well ("service", "server" as a few examples). Plus this heading is labeled 'Partner and Hybrid Cloud Infrastructure" - isn't that more than just storage?

Perhaps this heading should be more storage-centric, and then drop any non-storage terms (or move them to the general section if they don't already exist). Thoughts?

Reply:
@bergerhoffer This was a cross-product storage initiative, and the bulk of the products included are part of the HCI group post-reorg. I'm happy to rename this to "Storage" I think.